docs/docs/contributing/testing.md

```diff
@@ -21,63 +17,39 @@ These tests can be found in `<sub-project>/test` and are used to check
 functionality of specific parts of the codebase in isolation e.g: parsing,
 scanning and message errors.
 
-Running a single unit test class from sbt is as simple as:
+To run all tests in e.g., for the compiler test-suite you can write:
 
 ```bash
-> testOnly absolute.path.to.TestClass
+> dotty-compiler/test
 ```
 
-You can further restrict the executed tests to a subset of `TestClass` methods
-as follows:
+To run a single test class you use `testOnly` and the fully qualified class name. 
+For example:
 
 ```bash
-> testOnly absolute.path.to.TestClass -- *methodName
+> dotty-compiler/testOnly dotty.tools.dotc.transform.TreeTransformerTest
 ```
 
-## Integration tests
-These tests are Scala source files expected to compile with Dotty (pos tests),
-along with their expected output (run tests) or errors (neg tests).
-
-All of these tests are contained in the `./tests/*` directories.
-
-## scala-partest
-Historically these tests needed a structure which was generated by running the
-unit tests, and then that structure was in turn used by
-[scala-partest](http://github.com/scala/scala-partest) to run compilation tests
-in parallel.
-
-This test suite can still be used (and is currently a part of the CI to check
-that it has the same outcome as the new test suite). It is invoked from sbt by
-running one of the following commands:
+You can further restrict the executed tests to a subset of methods by appending ``-- *method_name`` 
+as in the example below:
 
 ```bash
-> partest-only-no-bootstrap
-> partest-only
-> partest
+> dotty-compiler/testOnly dotty.tools.dotc.transform.TreeTransformerTest -- *canOverwrite
```

Comment:
This could be written shorter as: `testOnly *.TreeTransformerTest -- *canOverride`, but I guess it's good to be consequential to what you decided to do WRT the above comment :)